Description: Remove unnecessary Requires: from gtk-sharp in tomboy-addins pcfile
Forwarded: not-needed
Origin: vendor
Author: Iain Lane <laney@ubuntu.com>

Index: tomboy.git/data/tomboy-addins.pc.in
===================================================================
--- tomboy.git.orig/data/tomboy-addins.pc.in	2010-01-09 14:21:26.260959442 +0000
+++ tomboy.git/data/tomboy-addins.pc.in	2010-01-09 14:21:30.890958275 +0000
@@ -5,6 +5,5 @@
 
 Name: Tomboy Addin Library
 Description: Library providing the interfaces for Tomboy addins
-Requires: gtk-sharp-2.0
 Version: @VERSION@
 Libs: -r:${libdir}/tomboy/Tomboy.exe
